src/__internal__/utils/helpers/tags/tags.ts

```diff
@@ -1,12 +1,21 @@
+interface Props {
```

Comment:
I think that name could be more meaningful, e.g. `DataProps`

Reply:
Fair point, I realised I'll need to extend the interface to support other unknown props as the function takes the `rest` props as the second argument